.github/workflows/new_contributor_pr.yml

```diff
@@ -8,7 +8,27 @@ permissions:
   pull-requests: write
 
 jobs:
+  check:
+    name: Close invalid PRs
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Close invalid PRs"
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const title = context.payload.pull_request.title;
+            const isInvalid = /99999/.test(title);
+            if (isInvalid) {
+              await github.rest.pulls.update({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                pull_number: context.payload.pull_request.number,
+                state: 'closed'
```

Comment:
Thought: Given people who create a toast PR are following the contributor guide, it might be nice to comment and push them back to the "Next steps" section so they know what to move on to: https://docs.djangoproject.com/en/5.2/intro/contributing/#next-steps. Rather than closing it without comment.